__init__.py

```diff
@@ -286,6 +286,7 @@ async def update_file(request):
 async def delete_file(request):
     data = await request.json()
     file_path = data['file_path']
+    delete_empty_folder = data['deleteEmptyFolder']
```

Comment:
do we have a default param for this? what if deleteEmptyFolder wasn't passed in. we should probably add some null value handling here for file_path and delete_empty_folder